src/lib/src/core/viewer-service/viewer.service.ts

```diff
@@ -118,12 +118,15 @@ export class ViewerService {
   }
 
   public home(): void {
+    if (!this.osdIsReady) {
```

Comment:
this.osdIsReady return a subject. Did you mean to check if osd is ready?

Reply:
Well spotted, that is what I meant!  I've updated it to check this.osdIsReady.getValue() instead.